modules/sharethroughBidAdapter.js

```diff
@@ -92,6 +92,26 @@ export const sharethroughAdapterSpec = {
       req.regs.ext.gpp_sid = bidderRequest.ortb2.regs.gpp_sid;
     }
 
+    if (bidderRequest?.ortb2?.regs?.ext?.dsa) {
```

Comment:
why are you shipping so much validation code on the object structure? why not just take whatever the publisher sets and validate it server side?

Reply:
@patmmccann  Our ad server team has requested we perform validation client-side.  It's an engineering/design decision.

Reply:
Please move the validation into a shared library; we're not allowing it here

Reply:
https://docs.prebid.org/dev-docs/modules/validationFpdModule.html actually we chatted and this seems like a better place

Reply:
@patmmccann - is a pub installing / using `validationFpdModule` with their installation enough to get validation for DSA, or are you saying that would be a good module to _add_ DSA validation to?

Reply:
We're saying runtime validation in your adapter isn't appropriate but your welcome to assist publishers in their validation efforts by commiting to this module